pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: breez_sdk
-description: A new flutter plugin project.
+description: Flutter bindings for the Breez SDK
+repository: https://github.com/breez/breez-sdk-flutter
```

Comment:
I think these changes have to be made in the SDK repo too, because the pubspec.yaml is committed from there.

Reply:
Good catch! Of course! Will update.

Reply:
Added here: https://github.com/breez/breez-sdk/pull/783